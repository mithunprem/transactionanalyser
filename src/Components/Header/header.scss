.transaction-analyser-header {
  background-color: #000000;
  min-height: 10vh;
  display: flex;
  justify-content: left;
  align-items: center;

  .header-text {
    color: #ffffff;
  }
}
