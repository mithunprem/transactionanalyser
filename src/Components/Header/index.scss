.transaction-analyser-header {
  background-color: #3a3636;
  height: 50px;
  display: flex;
  justify-content: left;
  align-items: center;

  .header-text {
    color: #ffffff;
  }
}
