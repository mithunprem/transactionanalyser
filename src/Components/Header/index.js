import React, { Fragment } from 'react';
import './header.scss';

const Header = () => {
  return (
    <Fragment>
      <header className="transaction-analyser-header">
        <h5 className="ml-3 header-text">Financial Transaction Analyser</h5>
      </header>
    </Fragment>
  );
}

export default Header;
