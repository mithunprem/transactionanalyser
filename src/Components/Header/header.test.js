import React from 'react';
import { create } from 'react-test-renderer';
import Header from '../../Components/Header';

describe("Header", () => {
  test("should render without throwing an error", () => {
    const header = create(<Header />).toJSON()
    expect(header).toMatchSnapshot()
  });
});
