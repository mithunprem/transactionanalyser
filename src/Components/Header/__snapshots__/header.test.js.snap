// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render without throwing an error 1`] = `
<header
  className="transaction-analyser-header"
>
  <h5
    className="ml-3 header-text"
  >
    Financial Transactions Analyser
  </h5>
</header>
`;
